spec/helpers/sufia_helper_spec.rb

```diff
@@ -100,13 +100,13 @@
     end
 
     context "when the user is on the dashboard page" do
-      it "defaults to My Files" do
+      it "defaults to My Works" do
         allow(helper).to receive(:params).and_return(controller: "dashboard")
         expect(helper.search_form_action).to eq(sufia.dashboard_files_path)
```

Comment:
OK, I see most of my comments boil down to "how far should this PR go?" Because I was wondering here whether `dashboard_files_path` should also be renamed to `dashboard_works_path`.